test/integration/narrow/server.test.js

```diff
@@ -15,8 +15,8 @@ describe('Application Startup Integration Test', () => {
     expect(server).toBeDefined()
     expect(server.info).toBeDefined()
 
-    await server.start()
-    expect(server.info.started).toBeGreaterThan(0)
+    await server.initialize()
+    expect(server.info.created).toBeGreaterThan(0)
```

Comment:
`started` property isn't set beyond `0` when `initialize` is used but `created` can be used to show the server instance was set.